### PR TITLE
Support: `HOT=0 yarn start` to disable HMR 

### PR DESCRIPTION
**Pull request by bencodeorg**
An explicit HOT=0 env variable will disable hot module reloading.

- `webpack-dev-server` is still enabled, and used to serve files via proxy AND writeToDisk: to update apps/build
- but the compiled JS will not contain hot reload hooks, so it shouldn't attempt to access port 9000

### CODE REVIEW

**Review by snickell — APPROVED** [review]
LGTM

File: apps/envConstants.js
Comment (bencodeorg) [inline apps/envConstants.js:41]: Gotta run home right now, but will this be false in our Gruntfile dev setup because we set process.env.HOT to 1 (number)? 🤔 Can check a bit later

Comment (snickell) [reply apps/envConstants.js:41]: process.env.* is always a string, just verified:
```
> process.env.boo
undefined
> process.env.boo = true
true
> process.env.boo
'true'
> 
```

**Comment by bencodeorg** [comment]
I can't approve as author, but LGTM if you want to merge before tomorrow :)

**Pull request merged**
